### ISSUE TITLE
TypeError: '<' not supported between instances of 'int' and 'datetime.datetime'

### ISSUE THREAD

**Issue by kkellner** [custom-components/alexa_media_player#611]
**Describe the bug**
At HA startup getting an error in log file:
`TypeError: '<' not supported between instances of 'int' and 'datetime.datetime'`

**System details**
 - Home-assistant (version): 0.107.7
 - Hassio (Yes/No): (Please note you may have to restart hassio 2-3 times to load the latest version of alexapy after an update. This looks like a HA bug).
 - alexa_media  (version from `const.py` or HA startup): 2.5.8
 - alexapy (version from `pip show alexapy` or HA startup):

**Additional context**

NOTE: changed s/n and other identifiers to xxxxx in below log.

```
2020-03-26 22:05:03 DEBUG (MainThread) [custom_components.alexa_media.sensor] value ('xxxxx', {'alarmTime': 1585325040000, 'automationId': None, 'createdDate': 1519416302707, 'deferredAtTime': None, 'deliveryType': None, 'deviceName': None, 'deviceSerialNumber': 'xxxxxx', 'deviceType': 'xxxxx', 'extensibleAttribute': None, 'geoLocationTriggerData': None, 'id': 'xxxxxx_REMINDER', 'lastOccurrenceTimeInMilli': 0, 'lastUpdatedDate': 0, 'musicAlarmId': None, 'musicEntity': None, 'notificationIndex': 'xxxxx-07d5-469c-b04c-bc54b39aafcf', 'originalDate': '2020-03-27', 'originalTime': '09:04:00.000', 'personProfile': None, 'provider': None, 'rRuleData': {'byMonthDays': None, 'byWeekDays': None, 'flexibleRecurringPatternType': 'ONCE_A_DAY', 'frequency': None, 'intervals': [1], 'nextTriggerTimes': None, 'notificationTimes': ['09:04:00.000'], 'recurEndDate': None, 'recurEndTime': None, 'recurStartDate': None, 'recurStartTime': None, 'recurrenceRules': ['FREQ=DAILY;BYHOUR=9;BYMINUTE=4;BYSECOND=0;INTERVAL=1;']}, 'recurrenceEligibility': False, 'recurringPattern': 'P1D', 'remainingTime': 0, 'reminderAppLabel': None, 'reminderLabel': 'Take your vitamins', 'reminderSubLabel': None, 'skillInfo': None, 'snoozedToTime': None, 'sound': {'displayName': 'Simple Alarm', 'folder': None, 'id': 'system_alerts_melodic_01', 'providerId': 'ECHO', 'sampleUrl': 'https://s3.amazonaws.com/deeappservice.prod.notificationtones/system_alerts_melodic_01.mp3'}, 'status': 'ON', 'targetPersonProfiles': None, 'timeZoneId': None, 'timerLabel': None, 'tmwNotificationProperties': None, 'triggerTime': 0, 'type': 'Reminder', 'version': '3189', 'date_time': '2020-03-27 09:04:00.000'})
2020-03-26 22:05:03 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up alexa_media platform for sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 179, in _async_setup_platform
    await asyncio.wait_for(asyncio.shield(task), SLOW_SETUP_MAX_WAIT)
  File "/usr/local/lib/python3.7/asyncio/tasks.py", line 442, in wait_for
    return fut.result()
  File "/config/custom_components/alexa_media/sensor.py", line 151, in async_setup_entry
    hass, config_entry.data, async_add_devices, discovery_info=None
  File "/config/custom_components/alexa_media/helpers.py", line 111, in wrapper
    result = await func(*args, **kwargs)
  File "/config/custom_components/alexa_media/sensor.py", line 118, in async_setup_platform
    account,
  File "/config/custom_components/alexa_media/sensor.py", line 467, in __init__
    client, n_json, "alarmTime", account, f"next {self._type}", "mdi:reminder"
  File "/config/custom_components/alexa_media/sensor.py", line 192, in __init__
    self._process_raw_notifications()
  File "/config/custom_components/alexa_media/sensor.py", line 200, in _process_raw_notifications
    self._all = list(map(self._update_recurring_alarm, self._all))
  File "/config/custom_components/alexa_media/sensor.py", line 253, in _update_recurring_alarm
    and alarm < dt.now()
TypeError: '<' not supported between instances of 'int' and 'datetime.datetime'
```

**Comment by alandtse** [comment]
Thanks. The bug report doesn't match the code that's [available for 2.5.8](https://github.com/custom-components/alexa_media_player/blob/dev/custom_components/alexa_media/sensor.py#L253).   
 Specifically, line 253 doesn't have `and alarm < dt.now() `.

Can you please uninstall and reinstall the component? Please confirm line 253 matches what I just linked.

**Comment by kkellner** [comment]
You linked the dev branch.   v2.5.8 matches:

https://github.com/custom-components/alexa_media_player/blob/v2.5.8/custom_components/alexa_media/sensor.py#L253

**Comment by alandtse** [comment]
Oh you're right. I forgot we hadn't pushed the fix yet.

**Comment by alandtse** [comment]
Dupe of #602.

**Comment by kkellner** [comment]
Confirmed fixed in v2.5.9
